### DISCUSSION TITLE
Veeam to Minio Testserver 8 Disks in Erasure Coding very poor Performance

### DISCUSSION THREAD

**Discussion by fsennj** [minio/minio#14317] [Q&A] [answered]
Hello All.

we offloaded from Veeam successfully to a Minio Testserver with 8 x 6TB SATA 7200 disks in Erasure Coding Mode. Until we hit the max. allowed restore points in Veeam everything was working fine and really fast (much much faster than the internet connection for upload was) but since Veeam needs to merge restore points to keep max. allowed quantity of restore points it does 30GB in 24hours which before it has done in like 30 mins..

Anyone any clue what that could be? Could some metadata from minio be placed on a NVME SSD to gain speed? thank you

**Comment by harshavardhana** [comment, marked as answer]
> Anyone any clue what that could be? Could some metadata from minio be placed on a NVME SSD to gain speed? thank you

Using HDDs with Veeam you will only get so much and you also gave MinIO only 8 drives - that's hardly any spindles to do anything useful for Veeam like workload.

**Reply by fsennj** [reply]
Good Morning - thank you for fast reaction. So increasing to 16 spindles could definately help? Any possibiltiy in moving an index / metadata to nvme drives?

**Reply by harshavardhana** [reply]
You need to fully use NVMe drives for all data with Veeam, HDDs won't work for Veeam workloads.

**Reply by fsennj** [reply]
OK thank you - then minio is not really useful as it is too expensive with big NVME articles.

**Reply by harshavardhana** [reply]
You need to use the right hardware for the application in question here.  Veeam is not suitable to be used with HDDs.

**Reply by fsennj** [reply]
yes understood but it is also a question of cost. What about hosters like Backblaze / Wasabi - as far as i have understood (correct me pleas if a m wrong) they are also using big HDDs. will i run there into the same problem?

**Reply by harshavardhana** [reply]
I cannot comment about other vendors.

**Reply by fsennj** [reply]
Right - thank you for all the information given.

**Reply by fsennj** [reply]
sorry one more question - upgrading to more affordable QLC SSD - is there any experience with that how this could work out? thank you

**Reply by harshavardhana** [reply]
QLC SSDs are fine @fsennj
